test/unit/core/selectors.ts

```diff
@@ -33,6 +33,47 @@ suite('selectors', ({ expect, stub }) => {
       expect(Selectors.isRefinementDeselected(<any>{}, 'my navigation', 4)).to.be.true;
     });
   });
+  describe('rangeLowest()', () => {
+    it('should return the rangeLowest', () => {
+      const rangeLowest = 4;
+      const id = 'hello';
+      const state: any = {
+        data: {
+          present: {
+            navigations: {
+              byId: {
+                [id]: {
```

Comment:
this structure doesn't need to be fully expanded like this, try collapsing a few layers in the middle to make it cleaner:

```js
const state: any = {
  data: {
    present: {
      navigations: {
        byId: { [id]: { rangeLowest } }
      }
    }
  }
};
```